langchain_tools_demo/tools.py

```diff
@@ -24,20 +24,13 @@
 from pydantic.v1 import BaseModel, Field
 
 BASE_URL = os.getenv("BASE_URL", default="http://127.0.0.1:8080")
-session = None
-
-
-# create a new client session
-async def get_session():
-    if session is None:
-        client_session = aiohttp.ClientSession()
-    return client_session
 
 
 # Helper functions
-async def get_request(url: str, params: dict) -> aiohttp.ClientResponse:
+async def get_request(
```

Comment:
You may be able to remove this function -- and add the bearer token (if it exists) on session creation

Reply:
removed